libs/jsonforms-components/src/lib/Controls/Inputs/InputBooleanControl.tsx

```diff
@@ -23,6 +23,10 @@ export const BooleanComponent = ({
   } as ControlProps);
 
   const text = label + (required ? ' (required)' : '');
+
```

Comment:
@solittlework can we refactor this section of code to check if isStepperReview.  It looks like some of these controls are the same?